lib/jwtool.js

```diff
@@ -0,0 +1,119 @@
+var jws = require('jws')
+var P = require('./promise')
+var request = require('request')
+var jwk2pem = require('pem-jwk').jwk2pem
+var pem2jwk = require('pem-jwk').pem2jwk
+
+function PrivateJWK(jwk) {
+  this.jwk = jwk
+  this.pem = jwk2pem(jwk)
+}
+
+PrivateJWK.fromPEM = function (pem, extras) {
+  return new PrivateJWK(pem2jwk(pem, extras))
+}
+
+PrivateJWK.prototype.toJSON = function () {
+  return this.jwk
+}
+
+PrivateJWK.prototype.sign = function (data) {
+  var payload = data || {}
+  payload.iss = this.jwk.iss
```

Comment:
Could it be useful to allow the input data to override `iss`?  I guess in theory it's tied to the private key, so maybe not...